<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="L-6063" ignored="1" name="Test" tests="11" failures="3" timestamp="04 Jun 2018 17:19:30 GMT" time="0.097" errors="0">
  <testcase name="testCalculateAppraisal" time="0.01" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic"/>
  <testcase name="testDependant" time="0.003" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic">
    <failure type="java.lang.AssertionError" message="expected [Dependant failed] but found [Dependant successful]">
      <![CDATA[java.lang.AssertionError: expected [Dependant failed] but found [Dependant successful]
at com.kpit.junit.examples.TestEmployeeBusinessLogic.testDependant(TestEmployeeBusinessLogic.java:55)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testDependant -->
  <testcase name="testException" time="0.003" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic"/>
  <testcase name="testYearlySalary" time="0.003" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic"/>
  <testcase name="testParameter" time="0.002" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic"/>
  <testcase name="testPrimeNumber" time="0.002" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.kpit.junit.examples.TestEmployeeBusinessLogic.testPrimeNumber(TestEmployeeBusinessLogic.java:84)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testPrimeNumber -->
  <testcase name="testPrimeNumber" time="0.001" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic"/>
  <testcase name="testPrimeNumber" time="0.001" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic"/>
  <testcase name="testPrimeNumber" time="0.001" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.kpit.junit.examples.TestEmployeeBusinessLogic.testPrimeNumber(TestEmployeeBusinessLogic.java:84)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testPrimeNumber -->
  <testcase name="testPrimeNumber" time="0.001" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic"/>
  <testcase name="testDependency" time="0.0" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic">
    <skipped/>
  </testcase> <!-- testDependency -->
  <testcase name="testSomething" time="0.0" classname="com.kpit.junit.examples.TestEmployeeBusinessLogic">
    <ignored/>
  </testcase> <!-- testSomething -->
</testsuite> <!-- Test -->
